cli/internal/promt.go

```diff
@@ -225,7 +225,7 @@ func ValidateDirForWrite(dir string) (bool, error) {
 }
 
 func ResolvePath(path string) (string, error) {
-	if path[:2] != "~/" {
+	if path[:1] != "~/" {
```

Comment:
Is there a reason for changing this? Don't know if this will be backward compatible change